OpenRA.Game/Widgets/Widget.cs

```
@@ -204,6 +206,8 @@ public Widget(Widget widget)
 			IgnoreChildMouseOver = widget.IgnoreChildMouseOver;
 			IgnoreMouseOver = widget.IgnoreMouseOver;
 
+			ChromeMetrics.TryGet<string>("DefaultCursor", out defaultCursor);
```

Should we throw an exception here if the get fails, or fall back to a default value? (The former is probably better, unless it is intentional that we ignore failure.)

---

Went with throwing for consistency.